src/components/views/spaces/SpacePublicShare.tsx

```diff
@@ -23,6 +23,7 @@ import {copyPlaintext} from "../../../utils/strings";
 import {sleep} from "../../../utils/promise";
 import {RoomPermalinkCreator} from "../../../utils/permalinks/Permalinks";
 import {showRoomInviteDialog} from "../../../RoomInvite";
+import {MatrixClientPeg} from "../../../MatrixClientPeg";
```

Comment:
Looks like the consensus seems to be with spaces around curlies.
Patentiently waiting for matrix-org/eslint-plugin-matrix-org#6/

Reply:
Sure, just was staying consistent with the rest of the file, if it was just a couple of wrongly styled imports I'd have replaced them all but leaving a mix is in-ideal

Reply:
(though this was probably an automatic import so should improve once the eslint rule lands)